frontend/e2e/cypress/support/index.js

```diff
@@ -46,6 +46,14 @@ Cypress.Commands.add('loginAsAdmin', () => {
   loginAsUser(TEST_ADMIN)
 })
 
+Cypress.Commands.add('loginAsRegisteredIndicatedUser', () => {
```

Comment:
What does registered *indicated* user mean, and how does it differ from the existing registered user?

Reply:
Only difference is that this user will be used for these tests only. This is because while using the other registered user, 'Donald', some subsequent test appeared broken, because of a extra group assignment. Now that 'Donald' is not used but 'Volodymyr', these can be tested and the tests don't break subsequent ones.

Reply:
Alright, I wonder if naming the function as "loginAsRegisteredUser2" would be more descriptive regarding the fact that there aren't many differences, but that might veer close to magic numbers. I'll trust your judgement on this one.